src/Rules/Data.hs

```diff
@@ -20,8 +20,9 @@ buildPackageData context@Context {..} = do
         cabalFile = pkgCabalFile package
         configure = pkgPath package -/- "configure"
         dataFile  = pkgDataFile context
+        setupConfigFile = pkgSetupConfigFile context
 
-    dataFile %> \mk -> do
+    [dataFile, setupConfigFile] &%> \(mk:setupConfig:_) -> do
```

Comment:
Alternatively, we should probably modify the line `build $ Target context GhcCabal [cabalFile] [mk]` by adding `setupConfig` into the list of output files.